src/main/java/uk/gov/ons/census/fwmt/fulfilment/service/FulfilmentService.java

```diff
@@ -43,13 +43,17 @@
   private MessagePublisher messagePublisher;
 
   public void processPauseCase(PauseOutcome pauseRequest, Instant messageReceivedTime) {
+    GatewayCache indCache = null;
     String caseId;
+    String individualCaseId = pauseRequest.getPayload().getFulfilmentRequest().getIndividualCaseId();
     String productCode = pauseRequest.getPayload().getFulfilmentRequest().getFulfilmentCode();
 
     final GatewayCache caseCache = cacheService.getByIdAndTypeAndExists(pauseRequest.getPayload().getFulfilmentRequest().getCaseId(),
         10, true);
-    final GatewayCache indCache = cacheService.getByIndividualCaseIdAndTypeAndExists(pauseRequest.getPayload().getFulfilmentRequest().getIndividualCaseId(),
-        10, true);
+
+    if (individualCaseId != null && !individualCaseId.equals("")) {
```

Comment:
use    !Strings.isEmpty(individualCaseId)

Reply:
That's been changed now.